editor/src/core/shared/element-template.tsx

```diff
@@ -320,6 +328,89 @@ export function clearJSXAttributeOtherJavaScriptUniqueIDs(
   }
 }
 
+export function simplifyAttributeIfPossible(attribute: JSXAttribute): JSXAttribute {
+  switch (attribute.type) {
+    case 'ATTRIBUTE_VALUE':
+    case 'ATTRIBUTE_OTHER_JAVASCRIPT':
+    case 'ATTRIBUTE_FUNCTION_CALL':
+      return attribute
+    case 'ATTRIBUTE_NESTED_ARRAY':
+      let simpleArray: Array<unknown> = []
+      let notSoSimpleArray: Array<JSXArrayElement> = []
+      let isSimpleArray: boolean = true
+      for (const elem of attribute.content) {
+        const simplifiedAttribute = simplifyAttributeIfPossible(elem.value)
+        switch (elem.type) {
+          case 'ARRAY_SPREAD':
+            notSoSimpleArray.push(jsxArraySpread(simplifiedAttribute, elem.comments))
+            break
+          case 'ARRAY_VALUE':
+            notSoSimpleArray.push(jsxArrayValue(simplifiedAttribute, elem.comments))
+            break
+          default:
+            const _exhaustiveCheck: never = elem
+            throw new Error(`Unhandled elem ${JSON.stringify(elem)}`)
+        }
+        if (isParsedCommentsEmpty(elem.comments) && isJSXAttributeValue(simplifiedAttribute)) {
```

Comment:
Very minor, but we could skip this if `isSimpleArray` is false

Reply:
Changed in 6fafae98.